Dalamud/Game/Gui/NamePlate/NamePlateInfoView.cs

```diff
@@ -0,0 +1,105 @@
+﻿using Dalamud.Game.Text.SeStringHandling;
+
+using FFXIVClientStructs.FFXIV.Client.UI;
+
+namespace Dalamud.Game.Gui.NamePlate;
+
+/// <summary>
+/// Provides a read-only view of the nameplate info object data for a nameplate. Modifications to
+/// <see cref="NamePlateUpdateHandler"/> fields do not affect this data.
+/// </summary>
+public interface INamePlateInfoView
+{
+    /// <summary>
+    /// Gets the displayed name for this nameplate according to the nameplate info object.
+    /// </summary>
+    SeString Name { get; }
+
+    /// <summary>
+    /// Gets the displayed free company tag for this nameplate according to the nameplate info object. For this field,
+    /// the quote characters which appear on either side of the title are NOT included.
+    /// </summary>
+    SeString FreeCompanyTag { get; }
+
+    /// <summary>
+    /// Gets the displayed free company tag for this nameplate according to the nameplate info object. For this field,
+    /// the quote characters which appear on either side of the title ARE included.
+    /// </summary>
+    SeString QuotedFreeCompanyTag { get; }
+
+    /// <summary>
+    /// Gets the displayed title for this nameplate according to the nameplate info object. For this field, the quote
+    /// characters which appear on either side of the title are NOT included.
+    /// </summary>
+    SeString Title { get; }
+
+    /// <summary>
+    /// Gets the displayed title for this nameplate according to the nameplate info object. For this field, the quote
+    /// characters which appear on either side of the title ARE included.
+    /// </summary>
+    SeString QuotedTitle { get; }
+
+    /// <summary>
+    /// Gets the displayed level text for this nameplate according to the nameplate info object.
+    /// </summary>
+    SeString LevelText { get; }
+
+    /// <summary>
+    /// Gets the flags for this nameplate according to the nameplate info object.
+    /// </summary>
+    int Flags { get; }
```

Comment:
How do you know what these mean? Does it make sense to expose the raw bitset like this as an int?